ui/hooks/useIsOriginalNativeTokenSymbol.js

```diff
@@ -18,12 +37,26 @@ export function useIsOriginalNativeTokenSymbol(chainId, ticker, type) {
           return;
         }
 
+        // exclude local dev network
+        if (isLocalhost(rpcUrl)) {
+          setIsOriginalNativeSymbol(true);
+          return;
+        }
+
         const mappedCurrencySymbol = CHAIN_ID_TO_CURRENCY_SYMBOL_MAP[chainId];
         if (mappedCurrencySymbol) {
           setIsOriginalNativeSymbol(mappedCurrencySymbol === ticker);
           return;
         }
 
+        const mappedAsNetworkCollision =
+          CHAIN_ID_TO_CURRENCY_SYMBOL_MAP_NETWORK_COLLISION[chainId];
```

Comment:
Would this approach work if there were multiple networks colliding with the same chain id?